persist/fs/seek_manager.go

```diff
@@ -282,11 +438,33 @@ func (m *seekerManager) seekersByTime(shard uint32) *seekersByTime {
 
 func (m *seekerManager) Close() error {
 	m.Lock()
-	defer m.Unlock()
+
```

Comment:
based on previosu comments, I'm gonna conver to defer m.Unlock() here

Reply:
actually nevermind, you end up holding the lock for way too long because of the openCloseLoopDoneCh